packages/open-truss/src/index.ts

```diff
@@ -1,3 +1,17 @@
 export { default as helloWorld } from './hello_world'
 export * from './utils/yaml'
 export * from './utils/format'
+class Foo {
```

Comment:
Are these classes left here intentionally?

Reply:
looks like @jonmagic removed those in his PR that branches of this https://github.com/open-truss/open-truss/pull/34/files#diff-f404b0ed66673ec723b8e71bf03bd51f76311501ead6c0f987804eac615dfd75L4.